app/Http/Controllers/ImportController.php

```diff
@@ -56,21 +56,23 @@ public function store(Request $request)
             throw new HttpException(500, 'Unable read and store file to S3.');
         }
 
-        if ($request->input('import-type') === 'turbovote') {
+        $type = $request->input('import-type');
+
+        if ($type === \Chompy\ImportType::$turbovote) {
```

Comment:
If you want, you can import this class at the top of the file:

```php
use Chompy\ImportType;
```

and then these can be shortened a wee bit, e.g:

```php
if ($type === ImportType::$turbovote) {
```